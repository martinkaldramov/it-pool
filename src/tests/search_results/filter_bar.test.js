import React from 'react';
import { shallow } from 'enzyme';
import FilterBar from '../../components/search_results/filter_bar';

describe('FilterBar', () => {
  const wrap = shallow(<FilterBar />);

  it('should render successfully', () => {
    expect(wrap).toMatchSnapshot();
  });
});
