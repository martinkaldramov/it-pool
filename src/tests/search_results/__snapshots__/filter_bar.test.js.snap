// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterBar should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FilterBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <p>
          Showing \`$
          286542
          \` results found
</p>,
        <FlatButton
          disabled={false}
          fullWidth={false}
          label="Filters"
          labelPosition="after"
          labelStyle={Object {}}
          onKeyboardFocus={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          primary={false}
          secondary={false}
/>,
        <ul
          className="active-filters"
>
          <li>
                    100+
          </li>
          <li>
                    &lt; 100
          </li>
          <li>
                    This week
          </li>
          <li>
                    Intemediate
          </li>
          <li>
                    Expert
          </li>
          <li>
                    Any Type
          </li>
</ul>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Showing \`$",
            286542,
            "\` results found",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Showing \`$",
          286542,
          "\` results found",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "disabled": false,
          "fullWidth": false,
          "label": "Filters",
          "labelPosition": "after",
          "labelStyle": Object {},
          "onKeyboardFocus": [Function],
          "onMouseEnter": [Function],
          "onMouseLeave": [Function],
          "onTouchStart": [Function],
          "primary": false,
          "secondary": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              100+
</li>,
            <li>
              &lt; 100
</li>,
            <li>
              This week
</li>,
            <li>
              Intemediate
</li>,
            <li>
              Expert
</li>,
            <li>
              Any Type
</li>,
          ],
          "className": "active-filters",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "100+",
            },
            "ref": null,
            "rendered": "100+",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "< 100",
            },
            "ref": null,
            "rendered": "< 100",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This week",
            },
            "ref": null,
            "rendered": "This week",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Intemediate",
            },
            "ref": null,
            "rendered": "Intemediate",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Expert",
            },
            "ref": null,
            "rendered": "Expert",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Any Type",
            },
            "ref": null,
            "rendered": "Any Type",
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <p>
            Showing \`$
            286542
            \` results found
</p>,
          <FlatButton
            disabled={false}
            fullWidth={false}
            label="Filters"
            labelPosition="after"
            labelStyle={Object {}}
            onKeyboardFocus={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            primary={false}
            secondary={false}
/>,
          <ul
            className="active-filters"
>
            <li>
                        100+
            </li>
            <li>
                        &lt; 100
            </li>
            <li>
                        This week
            </li>
            <li>
                        Intemediate
            </li>
            <li>
                        Expert
            </li>
            <li>
                        Any Type
            </li>
</ul>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Showing \`$",
              286542,
              "\` results found",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Showing \`$",
            286542,
            "\` results found",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "disabled": false,
            "fullWidth": false,
            "label": "Filters",
            "labelPosition": "after",
            "labelStyle": Object {},
            "onKeyboardFocus": [Function],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
            "onTouchStart": [Function],
            "primary": false,
            "secondary": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                100+
</li>,
              <li>
                &lt; 100
</li>,
              <li>
                This week
</li>,
              <li>
                Intemediate
</li>,
              <li>
                Expert
</li>,
              <li>
                Any Type
</li>,
            ],
            "className": "active-filters",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "100+",
              },
              "ref": null,
              "rendered": "100+",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "< 100",
              },
              "ref": null,
              "rendered": "< 100",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This week",
              },
              "ref": null,
              "rendered": "This week",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Intemediate",
              },
              "ref": null,
              "rendered": "Intemediate",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Expert",
              },
              "ref": null,
              "rendered": "Expert",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Any Type",
              },
              "ref": null,
              "rendered": "Any Type",
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
